### PR TITLE
Link the agent to the CDM Nessus server

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request adds Ansible code to this role that links the Nessus agent to the Nessus server in the CDM environment.

## 💭 Motivation and context ##

The Nessus agent and server will not communicate without this linking in place.

## 🧪 Testing ##

All `pre-commit` hooks and `molecule` tests pass.

## ✅ Checklist ##

* [x] This PR has an informative and human-readable title.
* [x] Changes are limited to a single goal - _eschew scope creep!_
* [x] All relevant type-of-change labels have been added.
* [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
* [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
* [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
* [x] Tests have been added and/or modified to cover the changes in this PR.
* [x] All new and existing tests pass.

### CODE REVIEW

**Review by dav3r — APPROVED** [review]
Looks good to me! 👍

**Review by hillaryj — APPROVED** [review]
🦕

File: tasks/main.yml [hunk 1]
```diff
@@ -62,3 +62,16 @@
   ansible.builtin.service:
     enabled: yes
     name: nessusagent
+
+- name: Link the Nessus Agent to the CDM Nessus server
+  ansible.builtin.command:
+    argv:
+      - /opt/nessus_agent/sbin/nessuscli
```
Comment (mcdonnnj) [inline tasks/main.yml]: Should this use the value of `install_directory`?

File: tasks/main.yml [hunk 2]
```diff
@@ -62,3 +62,16 @@
...
+      - agent
+      - link
+      - --key={{ key }}
+      - --host={{ hostname }}
+      - --port={{ port }}
+      - --groups={{ nessus_groups }}
+      - --offline-install
+    creates: /opt/nessus_agent/var/nessus/master.key
```
Comment (mcdonnnj) [inline tasks/main.yml]: Same question here.

**Review by mcdonnnj — APPROVED** [review]
Makes sense, but I did have a question about two hard-coded paths.

File: tasks/main.yml (hunk 1 above)
Comment (jsf9k) [reply tasks/main.yml]: See commit 1bbe524.

File: tasks/main.yml (hunk 2 above)
Comment (jsf9k) [reply tasks/main.yml]: See commit 1bbe524.

**Review by mcdonnnj — APPROVED** [review]

File: tasks/main.yml (hunk 1 above)
Comment (mcdonnnj) [reply tasks/main.yml]: Thanks for taking care of that!

**Review by hillaryj — APPROVED** [review]
👍🏻

**Pull request merged**